src/iosrv/srv.c

```diff
@@ -266,6 +268,24 @@ check_sleep_list()
 	}
 }
 
+/**
+ * sleep micro seconds, then being rescheduled.
+ * \param[in]	us	micro seconds to sleep for
+ */
+int
+dss_sleep(uint64_t sleep_usec)
```

Comment:
Defect: The original dss_sleep() took millisecond, you changed it to microsecond, so all callers need to be changed as "dss_sleep(N * 1000)?

Reply:
ah, right.